src/index.html

```diff
@@ -7,12 +7,72 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
The href attribute points to a '.scss' file, which is a Sass file. HTML cannot directly use Sass files; they need to be compiled to CSS first. Ensure that you compile your Sass file to a CSS file and update the href to point to the resulting '.css' file.